thavalon-webapp/src/utils/GameSocket.tsx

```diff
@@ -0,0 +1,67 @@
+enum WEBSOCKET_READYSTATE {

```

Comment:
I think there is a prebuilt enum for this. Might be worth a Google, because I forget the name of it, but I think these statuses are in base Typescript.

Reply:
you are correct, at https://developer.mozilla.org/en-US/docs/Web/API/WebSocket